### PR TITLE
97 bug name esb

### LINKED ISSUE #97: Fix NameElementSelectionBehaviours
- [ ] PR owner
- [ ] Participant
- [ ] Other contributor (notified in a comment, but not a participant yet)

### PR DESCRIPTION
**Pull request by MathiasMeuleman**
Will close #97 

Added composedSelectors for PR_CREATOR, PR_PRATICIPANT and PR_OTHER_CONTRIBUTOR.
Things now get selected :)

### CODE REVIEW

**Comment by thervh70** [comment]
The diff size is humongous haha

File: src/octopeer-github/content/ElementSelectionBehaviour/ElementSelectionBehaviourData.ts [hunk 1]
```diff
@@ -139,22 +139,36 @@ const elementSelectionBehaviourDataList: ElementSelectionBehaviourData[] = [
         selector: "a.message",
     },
     {
+        composedSelector: () => {
+            let prContributers: string[] = [];
+            $(".participation-avatars a").each(function() { prContributers.push($(this).attr("aria-label")); });
+            // Substring(1) is used because of the @ prefix
```
Comment (thervh70) [inline src/octopeer-github/content/ElementSelectionBehaviour/ElementSelectionBehaviourData.ts]: Useful inline comment here. +1

**Comment by thervh70** [comment]
All new ESBs have a duplicate selector (both `selector` and `compoundselector`. Can't you change this duplication?

**Comment by MathiasMeuleman** [comment]
@thervh70 When also making selector optional, both selector and composedSelector can be left empty, which is not something you would want

**Comment by mpsijm** [comment]
I agree with Mathias on that point :)

**Comment by mpsijm** [comment]
This is ready for merging, if Robin also agrees :)

**Comment by thervh70** [comment]
Good! I will merge this.

**Pull request merged**